python/rpdk/java/templates/init/guided_aws/StubCreateHandler.java

```diff
@@ -0,0 +1,131 @@
+package {{ package_name }};
+
+import java.util.Objects;
+import software.amazon.awssdk.awscore.AwsRequest;
+import software.amazon.awssdk.awscore.AwsResponse;
+import software.amazon.cloudformation.exceptions.CfnAlreadyExistsException;
+import software.amazon.cloudformation.exceptions.CfnInvalidRequestException;
+import software.amazon.cloudformation.exceptions.CfnNotFoundException;
+import software.amazon.cloudformation.exceptions.CfnResourceConflictException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class {{ operation }}Handler extends BaseHandlerStd {
+    private static final String CREATED_RESOURCE_STATUS = "available";
+    private Logger logger;
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+        final AmazonWebServicesClientProxy proxy,
+        final ResourceHandlerRequest<ResourceModel> request,
+        final CallbackContext callbackContext,
+        final ProxyClient<ServiceSdkClient> proxyClient,
+        final Logger logger) {
+
+        this.logger = logger;
+
+        final ResourceModel model = request.getDesiredResourceState();
+
+        // TODO: Adjust Progress Chain according to your implementation
+
+        return ProgressEvent.progress(model, callbackContext)
+            .then(progress -> checkForPreCreateResourceExistence(proxy, request, progress))
+            .then(progress ->
+                // If your service API throws 'ResourceAlreadyExistsException' for create requests then CreateHandler can return just proxy.initiate construction
+                proxy.initiate("Service-Name::{{ operation }}-Custom-Resource", proxyClient, model, callbackContext)
```

Comment:
I was assuming this code-gen will be build-able so that the subsequent infrastructure can be set up as well. That would mean the actual implementation will be commented out with explanation on each API/helper methods defined. and meanwhile the handler will simply return a success event.

Reply:
good point, yea it's more like a guidelines at this point.